barcodes_generator_location/models/barcode_rule.py

```diff
@@ -0,0 +1,11 @@
+# Copyright 2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields, models
+
+
+class BarcodeRule(models.Model):
+    _inherit = 'barcode.rule'
+
+    generate_model = fields.Selection(
+        selection_add=[('stock.location', 'Stock Location')],
+    )
```

Comment:
Maybe you need create an uninstall hook for set this field to other or blank...

Reply:
@chienandalu please take into account this remark.

Reply:
I opted for delete them directly